lib/src/sprite_batch.dart

```diff
@@ -212,23 +208,23 @@ class SpriteBatch {
   ///
   /// You can transform the sprite from its [offset] using [scale], [rotation] and [anchor].
   ///
-  /// The [color] parameter allows you to render a color behind the batch item, as a background color.
+  /// The [color] paramater allows you to render a color behind the batch item, as a background color.
```

Comment:
The previous one was correct right?

Reply:
I didn't change this, it was probably fixed on master and not rebased